week-1.py

```diff
@@ -71,7 +77,7 @@ def playHand(players):
     for player in players:
         dealerCard = random.choice(cards)
         #[EXECUTE THE PLAY() FUNCTION FOR EACH PLAYER USING THE DEALER CARD, AND PRINT OUT THE RESULTS]
-        
+        return player.play(random.choice(cards))
```

Comment:
you should use 'print' here instead of 'return' to print the results of the play() function for each hand.